test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -14,7 +14,13 @@ public class CompatibilityTests
         [InlineData("dotnet", "dotnet", true)]
         [InlineData("dotnet5.1", "dotnet", true)]
 
-        // dnxcore50 -> dotnet
+        // dnxcore50 -> dotnet, netstandard
+        [InlineData("dnxcore50", "netstandard1.5", true)]
```

Comment:
Per @ericstj's [change](https://github.com/NuGet/NuGet.Client/commit/a15e8ff0c6de85eb8da995672ce4bb1751fc141d#diff-466ac121bd03ff666f4f494225859d8eL324), `dnxcore50` supports `dotnet5.6`. We are saying that anything that supports `dotnet5.6` also supports `netstandard1.5`, I tested this.